integration-tests/_tests/12-auth-create-user.test.ftd

```diff
@@ -0,0 +1,85 @@
+;; require fastn built using --features auth flag
+;; FASTN_DB_URL, FASTN_ENABLE_AUTH must be set
+-- import: fastn
+
+-- fastn.test: create user
+
+-- fastn.post: Create user with insufficient data should fail
```

Comment:
I think we should rename these to `fastn.view` and `fastn.action`. Also for actions that are failing we can create `fastn.failing-action`, so we do not have to explicitly assert error code etc. We can even assert error messages:

```ftd
-- fastn.failing-action: Create user with insufficient data should fail
url: /-/auth/create-user/
email-error: Email is required

{
    "username": "siddhant"
}
```

Or even better:

-- fastn.failing-action: Create user with insufficient data should fail
url: /-/auth/create-user/
username: siddhant
email-error: Email is required
```

Reply:
We can leave `fastn.get` and `fastn.post` and these new helpers `fastn.view`, etc.

Reply:
In another PR.